sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/implementation/models/PathUpdateHeaders.java

```diff
@@ -107,6 +107,18 @@
     @JsonProperty(value = "x-ms-properties")
     private String properties;
 
+    /*
+     * When performing setAccessControlRecursive on a directory, the number of
+     * paths that are processed with each invocation is limited.  If the number
+     * of paths to be processed exceeds this limit, a continuation token is
+     * returned in this response header.  When a continuation token is returned
+     * in the response, it must be specified in a subsequent invocation of the
+     * setAccessControlRecursive operation to continue the
+     * setAccessControlRecursive operation on the directory.
+     */
+    @JsonProperty(value = "x-ms-continuation")
+    private String xMsContinuation;
```

Comment:
This may be more of a swagger thing but should this generate as continuation rather than xMsContinuation?